src/components/card/Card.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { ReactNode } from 'react';
+import './card.css';
+
+interface CardProps {
+  children?: ReactNode;
+  widthSize: string;
+  heightSize: string;
+  className?: string;
+}
+
+export const Card = ({
+  children,
+  widthSize,
+  heightSize,
+  className,
+}: CardProps) => {
+  return (
+    <div
```

Comment:
dont bother with all the card imgs or style , Card component is just a div with a box-shadow and custom size and recive a children delete unncesary code css/assest/ingnore the figma desing the content was just a example just creatre 3 stories small,medium and larga but with a children and thats it , let me know if you have any questions